src/core/pageContent/pageContentLayout.module.scss

```diff
@@ -45,19 +45,28 @@
       flex: calc((100% / 3) * 2) 0 1;
       max-width: calc((100% / 3) * 2);
       // TODO: Replace with flex-gap when browsers support it
-      margin-right: $spacing-xs;
+      margin-right: 22px;
```

Comment:
Shouldn't these be in some configurable variable like they were earlier? Or does the 22px really fit in every app that are going to use this? Could the 22px be set as a default value but be used as a variable?